src/server.py

```diff
@@ -28,12 +27,15 @@ class HackSoundPlayer(GObject.Object):
         'error': (GObject.SignalFlags.RUN_FIRST, None, (GLib.Error, str))
     }
 
-    def __init__(self, uuid_, metadata, sender, metadata_extras=None):
-        GObject.Object.__init__(self)
+    def __init__(self, bus_name, sound_event_id, uuid_, metadata,
+                 metadata_extras=None):
+        super().__init__()
+        self.logger = Logger(PlayerFormatter, self)
+        self.bus_name = bus_name
```

Comment:
Sorry that I keep noticing new things in this pull request. I noticed in this case that we are replacing `self.sender` with `self.bus_name` but nothing actually changes in the body of this class. That seems to me not ideal, because someone will think it's not used and remove it.

Would it be possible instead of having PlayerFormatter and ServerFormatter, to make a method in HackSoundPlayer and HackSoundServer called format_log? That way, ObjectFormatter can call `self.obj.format_log(self, ...)` and we also don't have to have this boolean flag for whether to show the object's address or not. And then bus_name and sound_event_id can be private attributes.

Reply:
I don't think this is ideal since this would pass all the code of the ServerFormatter and PlayerFormatter to the HackSoundServer and HackSoundPlayer, respectively. I don't want to have code relating to logging in the HackSoundServer or HackSoundPlayer classes to be honest, because this is unrelated. If we want to avoid to delete `self.bus_name`, then a comment is enough.

Reply:
Let me know what you think about this.